src/search/landmarks/landmark_factory_rpg_sasp.cc

```diff
@@ -176,6 +177,8 @@ void LandmarkFactoryRpgSasp::found_simple_lm_and_order(
         return;
     }
 
+    vector<FactPair> fact = {a};
+    Landmark landmark(fact, false, false);
```

Comment:
Again, I'd prefer `facts` (or just use `{a}` in-place).

Reply:
We did change this.